irohad/ordering/impl/ordering_service_impl.cpp

```diff
@@ -44,19 +46,21 @@ namespace iroha {
     }
 
     void OrderingServiceImpl::generateProposal() {
-      auto txs = decltype(std::declval<model::Proposal>().transactions)();
-      for (model::Transaction tx;
-           txs.size() < max_size_ and queue_.try_pop(tx);) {
-        txs.push_back(std::move(tx));
+      std::vector<shared_model::proto::Transaction> poped_tx;
```

Comment:
Not sure if popped is a great choice here, maybe dequeued?

Reply:
There's no deque in tbb, containers from standard lib don't support concurrency